hal/network/lwip/ppp_client.cpp

```diff
@@ -31,6 +31,10 @@ extern "C" {
 #include "system_error.h"
 #include "lwiplock.h"
 
+#undef LOG_COMPILE_TIME_LEVEL
+#define LOG_COMPILE_TIME_LEVEL LOG_LEVEL_ALL // include errors
+LOG_SOURCE_CATEGORY("net.ppp.client")
```

Comment:
Better to add `;` so that some editors/viewers e.g. Github don't get confused.

Reply:
Pretty sure I just copied this from one of the other files but I’m happy to add ;

Reply:
Yeah, we have a lot of places where we don't have it, but you can see in the diff that it messes things up.

Reply:
I typically do use that convention with macros, you can never have enough semicolons in C/C++ I’ll fix it in the morning.

Reply:
Added ;